app/_locales/en/messages.json

```
@@ -878,6 +878,10 @@
   "infoHelp": {
     "message": "Info & Help"
   },
+  "infuraBlockedNotification": {
+    "message": "MetaMask is unable to connect to the blockchain host. Review possible reasons $1.",
+    "description": "$1 is a clickable link."
```

The most important thing about `$1` for translators to understand is that `$1` will have the text "here" as a link.
```suggestion
    "description": "$1 is a clickable link with with text defined by the 'here' key"
```

---

suggested change implemented